tests/helpers.py

```diff
@@ -2,41 +2,11 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import datetime
 from uuid import uuid4
 
 from celery.result import AsyncResult
 
 
 class SimpleFakeTaskBase(object):
-    def apply_async(self, args=None, kwargs=None, task_id=None, producer=None, link=None, link_error=None, **options):
+    def apply_async(self, *args, **kwargs):
```

Comment:
The reason I made the `apply_async` signature the way it is was to keep aligned with the Celery task `apply_async` signature.  Do you think it's worth keeping the unmodified version, or do you still think it's more appropriate to simply accept `*args, **kwargs`?